spec/features/articles_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :feature do
+  it "Lets you create an article" do
+    visit new_article_path
+
+    fill_in "Title:", with: "ArticleTest"
+    fill_in "Content:", with: "This is content for this article"
+
+    click_on 'Create'
+
+    expect(page).to have_content "ArticleTest"
+  end
+
+  context 'with existing articles'do
+    before do
+      Article.create(title: 'First Article', body: 'First Article body')
+      Article.create(title: 'Second Article', body: 'Second Article body')
```

Comment:
this looks good for now, when you eventually start creating records that you want to interact with more directly (maybe implement an edit page and want to go to the edit page for a specific article), take a look at pulling in FactoryBot to make resource creation easier - https://github.com/thoughtbot/factory_bot_rails